index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="The perfect personal assistant. Relaxr does your job for you so you can enjoy lift as it is meant to be.">
+    	<meta name="keywords" content="relaxr, automation, personal">
+
+		<title>Relaxr Landing Page</title>
+
+        <link rel="stylesheet" href="css/relaxr.css">
+        <!-- Whenever I include the normalize and reset stylesheets, my css files look off. Please provide feedback! -->
+        <!-- <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/reset.css"> -->
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+
+	</head>
+	
+	<body>
+        <header>
+            <div id="header">
+                <h1>Relaxr</h1>
+                <h2>Get piece of mind with a single tap</h2>
+                <button class="btn-get-it-now">Get it now</button>
+            </div>
+        </header>
+
+        <div id="benefits">
+            <h2>Benefits</h2>
+            <p>The perfect personal assistant. Relaxr does your job for you so you can enjoy lift as it is meant to be.</p>
+            <ul>
+				<li>Schedule meetings for you</li>
```

Comment:
@srhjang the <li> tags don't need to be indented that many times on under the <ul>